addons/website/static/tests/tours/focus_blur_snippets.js

```diff
@@ -0,0 +1,79 @@
+odoo.define(function (require) {
+'use strict';
+
+const ajax = require('web.ajax');
+const tour = require('web_tour.tour');
+
+const blockIDToData = {
+    parent: {
+        selector: '.s_focusblur',
+        name: 'section',
+        number: 0,
+    },
+    child1: {
+        selector: '.s_focusblur_child1',
+        name: 'first child',
+        number: 1,
+    },
+    child2: {
+        selector: '.s_focusblur_child2',
+        name: 'second child',
+        number: 2,
+    },
+};
+
+function clickAndCheck(blockID, expected) {
+    const blockData = blockIDToData[blockID] || {};
+
+    return [{
+        trigger: blockData.selector || '#wrapwrap',
+        content: blockID ? `Enable the ${blockData.name}` : 'Disable all blocks',
+    }, {
+        trigger: blockID
+            ? `.oe_overlay.ui-draggable:eq(${blockData.number}).oe_active`
+            : `#oe_manipulators:not(:has(.oe_active))`,
+        content: 'Once the related overlays are enabled/disabled, check that the focus/blur calls have been correct.',
+        run: function (actions) {
+            const result = window.focusBlurSnippetsResult;
+            window.focusBlurSnippetsResult = [];
+
+            if (expected.length !== result.length
+                    || !expected.every((item, i) => item === result[i])) {
+                console.error(`
+                    Expected: ${expected.toString()}
+                    Result: ${result.toString()}
+                `);
+            }
+        },
+    }];
+}
+
+window.focusBlurSnippetsResult = [];
+
+tour.register('focus_blur_snippets', {
+    url: '/?enable_editor=1',
+}, [
+    {
+        trigger: 'body',
+        run: function () {
+            ajax.loadJS('/website/static/tests/tours/tour_utils/focus_blur_snippets_options.js').then(function () {
+                $('body').addClass('focus_blur_snippets_options_loaded');
+            });
+        }
+    },
+    {
+        trigger: '#snippet_structure .oe_snippet:has(.oe_snippet_body.s_focusblur) .oe_snippet_thumbnail',
+        extra_trigger: 'body.focus_blur_snippets_options_loaded',
+        content: 'Drag the custom block into the page',
```

Comment:
weird to not have content first IMO

Reply:
I think there are more examples in Odoo with the trigger first (always did it like that when we introduced it 6 years ago)... but agree, better first :+1:

Reply:
Hm that's weirdly true, I had a quick look.
Almost every single frontend module tours are defined with `content` first while it is the opposite for backend modules (sadly, because when you quickly read a tour in its JS file, you just want to read all the `content` entries first to figure what the tour does, having it first is a lot easier).
But since you agree it is better like that, nothing to debate :D .. :+1: